onnx/backend/test/report/coverage.py

```diff
@@ -159,7 +159,7 @@ def report_csv(self, all_ops, passed, experimental):  # type: (List[Text], List[
         if os.path.isfile(nodes_path):
             with open(nodes_path, 'r') as nodes_file:
                 reader = csv.DictReader(nodes_file)
-                frameworks = list(reader.fieldnames)
+                frameworks = list(reader.fieldnames)  # type: ignore
```

Comment:
The error is `Argument 1 to "list" has incompatible type "Optional[Sequence[str]]"; expected "List[str]"`.
I tried to make `reader.fieldnames` as `List Optional[Sequence[str]]` or others, but it still showed error anyway. Please let me know if there is a better way to solve it. Thanks!

Reply:
The problem is that `reader.fieldnames` may be None. We can assert that it's not to avoid the type error:

```suggestion
                assert reader.fieldnames
                frameworks = list(reader.fieldnames)
```

Reply:
Nice trick. I have updated and it worked. Thank you for the suggestion!